full-report/action.yml

```diff
@@ -7,9 +7,16 @@ inputs:
   release_version:
     description: The published version
     required: true
+  sarif_report_target_ref:
+    description: The git ref (usually the target release branch) to pull sarif reports from.
+    required: true
```

Comment:
This would technically require us to tag `v3` - any way we can default this to empty and use the version name as fall back?

Reply:
I started a conversation on slack about whether breaking changes are acceptable.  But I'll update this regardless

Reply:
I agree we should keep backwards compat as a matter of practice unless we explicitly want to move to `v3`.